services/iam.py

```diff
@@ -2,22 +2,74 @@
 import boto3
 
 
-# client = boto3.client("")
+client = boto3.client("iam")
 
 
 def iam_policy_no_statements_with_admin_access():
+    compliant_resource = []
+    non_compliant_resources = []
+    policies = client.list_policies(Scope="Local")["Policies"]
+
+    for policy in policies:
+        policy_version = client.get_policy_version(PolicyArn=policy["Arn"], VersionId=policy["DefaultVersionId"])[
+            "PolicyVersion"
+        ]
+
+        if "'Effect': 'Allow', 'Action': '*', 'Resource': '*'" not in str(policy_version["Document"]):
+            compliant_resource.append(policy["Arn"])
+        else:
+            non_compliant_resources.append(policy["Arn"])
```

Comment:
IAM policy를 작성하는 방식에 따라서 단순 string match로 검사할 수 없는 경우가 있습니다. Attribute를 하나씩 확인할 필요가 있습니다.
```suggestion
    for policy in policies:
        policy_version = client.get_policy_version(PolicyArn=policy["Arn"], VersionId=policy["DefaultVersionId"])[
            "PolicyVersion"
        ]

        for statement in policy_version["Document"]["Statement"]:
            if (
                statement["Action"] == "*"
                and statement["Resource"] == "*"
                and statement["Effect"] == "Allow"
            ):
                non_compliant_resources.append(policy["Arn"])
                break
        else:
            compliant_resource.append(policy["Arn"])
```